src/hooks/useCart/useCart.tsx

```diff
@@ -235,11 +236,22 @@ export const useCart = (
    */
   const _completeCheckout = useCallback(() => {
     const complete = async (): Promise<void> => {
-      const transactions = Object.values(cart)
+      const allCleanedIdentifierInputs: IdentifierInput[] = [];
+      let transactions: any = [];
```

Comment:
do we still need to declare the transactions here? or can we just use `const transactions` in the next line?

Reply:
fixed!